api/proto/disperser/disperser.proto

```diff
@@ -159,8 +159,8 @@ enum BlobStatus {
 	// INSUFFICIENT_SIGNATURES means that the confirmation threshold for the blob was not met
 	// for at least one quorum.
 	INSUFFICIENT_SIGNATURES = 5;
-
-	// DISPERSING means that the blob is currently being dispersed to DA Nodes and being confirmed onchain
+	// DISPERSING means that the blob is currently being dispersed to DA Nodes, 
+	// and the batcher is gathering their signatures.
```

Comment:
Between DISPERSING and its next state (InsufficientSig, Confirmed, or failed), there are two phases
1. Dispersing to DA nodes and collecting signature
3. submitting the transaction on chain and waiting for tx receipt

Reply:
my bad, not very accurate in the first comment

Reply:
[1588266](https://github.com/Layr-Labs/eigenda/pull/828/commits/15882662fd15d415bf0aab2e2610b48e49400f92)